integration/hyva/homepage.spec.js import path to cart.json fixture is wrong

**Describe the bug**
```
Running:  homepage.spec.js
Oops...we found an error preparing this test file:
  > cypress/integration/hyva/homepage.spec.js
The error was:
Error: Webpack Compilation Error
./cypress/integration/hyva/homepage.spec.js
Module not found: Error: Can't resolve '../../../fixtures/hyva/selectors/cart.json' in '/data/web/public/cypress/integration/hyva'
```

**To Reproduce**
Steps to reproduce the behavior:
1. Run the tests in the current main branch head

**Expected behavior**
Test can compile

**Additional context**
The path to the `cart.json` has one `../` to many at https://github.com/elgentos/magento2-cypress-testing-suite/blob/main/cypress/integration/hyva/homepage.spec.js#L6

--- a/cypress/integration/hyva/cms/cms-pages.spec.js
+++ b/cypress/integration/hyva/cms/cms-pages.spec.js
@@ -1,5 +1,5 @@
-import selectors from "../../../../fixtures/hyva/selectors/cms.json";
-import cms from "../../../../fixtures/hyva/cms.json";
+import selectors from "../../../fixtures/hyva/selectors/cms.json";
+import cms from "../../../fixtures/hyva/cms.json";
 
 describe('CMS tests', () => {
   it('Can display the default 404 page', () => {

--- a/cypress/integration/hyva/homepage.spec.js
+++ b/cypress/integration/hyva/homepage.spec.js
@@ -3,7 +3,7 @@ import selectors from '../../fixtures/hyva/selectors/homepage.json';
 import searchSelectors from '../../fixtures/hyva/selectors/search.json';
 import product from '../../fixtures/hyva/product.json';
 import account from '../../fixtures/account.json';
-import cart from "../../../fixtures/hyva/selectors/cart.json";
+import cart from "../../fixtures/hyva/selectors/cart.json";
 
 describe('Home page tests', () => {
     beforeEach(() => {

